sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/sm_pipeline_with_amazon_forecast.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>The two raw cells with IAM policies do not show up on GitHub web interface. I think they should be markdown rather than raw?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>In markdown the cells become unformatted and hard to read. We copied this structure from the notebook under https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-pipelines/nlp/amazon_comprehend_sagemaker_pipeline/sm_pipeline_with_comprehend.ipynb. We could put the cells as code (but that might be somewhat confusing) OR upload pictures into the notebook. Whichever you prefer.</p>

Reply:
<p>In the most recent commit we put it as code.</p>